packages/arb-token-bridge-ui/src/util/ArbTokenUtils.ts

```diff
@@ -0,0 +1,10 @@
+import { ERC20BridgeToken } from '../hooks/arbTokenBridge.types'
+import { SPECIAL_ARBITRUM_TOKEN_TOKEN_LIST_ID } from './TokenListUtils'
+
+export function isArbitrumToken(token: ERC20BridgeToken | null) {
```

Comment:
Do we need a new file for this? Could use `TokenUtils`

Reply:
i wanted to, but looked at the failed e2e.... `TokenUtils` was imported into `synpress.config.ts` and from `synpress` it couldn't resolve the `@images/` paths, so i changed the paths back to relative paths, but then svg can't be processed there

too much effort to fix it just because this file imported `TokenUtils` :D

<img width="967" alt="image" src="https://github.com/OffchainLabs/arbitrum-token-bridge/assets/13184582/ca78c16d-ea5c-4356-9980-d98b5312b574">